Include/internal/pycore_stackref.h

```diff
@@ -237,6 +238,44 @@ _PyObjectStack_FromStackRefStack(PyObject **dst, const _PyStackRef *src, size_t
     }
 }
 
+// StackRef type checks
+
+static inline bool
+PyStackRef_CheckTupleExact(_PyStackRef stackref)
```

Comment:
I think `PyStackRef_TupleCheckExact` would be more consistent with the other names you've chosen.

Reply:
It wasn't actually used, so I've deleted it